### DISCUSSION TITLE
Today's activity (commits) is being shown in yesterday's activity.

### DISCUSSION THREAD

**Discussion by UmerGillani36** [community/community#58607] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Today, I committed and pushed my code, but in the progress table, today's commits are being shown under yesterday's commits, and today's progress is empty.

**Comment by airtower-luna** [comment]
Two things to look at:

* The day boundaries should be based on your local timezone. If that's set/reported incorrectly, commits may end up on the wrong day.
* Commits will be shown with the committer time & date, not the time & date of the push.

**Reply by UmerGillani36** [reply]
The streak has been updated but is not reflected in the progress table. What should I do now?

**Comment by Aliyan-Ansari** [comment, marked as answer]
you need to set your date and time because it reflects on github from you machine time
